packages/css-framework/src/components/_tab-set.scss

```diff
@@ -0,0 +1,49 @@
+$active-border-color: #2a77C7;
+
+.rn-tab-set {
+  display: flex;
+  flex-flow: column;
+}
+
+.rn-tab-set__head {
+  border-bottom: 1px solid #e2e9ee;
+}
+
+.rn-tab-set__tabs {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.rn-tab-set__tab-item {
+  display: inline-block;
+  border-bottom: 4px solid transparent;
+  color: #748999;
+
+  &.is-active {
+    color: #3e5667;
+    border-color: $active-border-color;
+  }
+}
+
+.rn-tab-set__tab {
+  margin: 0 spacing(1);
+  padding: spacing(4) spacing(8);
+  border: none;
+  font-size: font-size(xs);
+  font-weight: 700;
```

Comment:
Starting to think we need vars for weight

Reply:
Sounds good to me - here i've used the value that most closely reflected the screenshots in the Jira ticket.

Reply:
Lets discuss this on Wednesday - Designers have already mentioned the need to do a proper review the CSS framework.